azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -46,16 +45,28 @@ public KeyVaultEnvironmentPostProcessorHelper(final ConfigurableEnvironment envi
         sendTelemetry();
     }
 
-    public void addKeyVaultPropertySource() {
-        final String vaultUri = getProperty(this.environment, Constants.AZURE_KEYVAULT_VAULT_URI);
+    /**
+     * Add a key vault property source.
+     *
+     * <p>
+     * The normalizedName is used to target a specific key vault (note if the
+     * name is the empty string it works as before with only one key vault
+     * present).
+     * </p>
+     *
+     * @param postfix the postfix.
```

Comment:
inaccurate parameter name `postfix`

Reply:
Changed it to `normalizedName` and added a more detailed explanation of what the normalizedName is.